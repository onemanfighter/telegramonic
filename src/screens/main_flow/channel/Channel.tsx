const Channel = () => {
  return (
    <div>
      <h1>Channel</h1>
    </div>
  );
};

export default Channel;
