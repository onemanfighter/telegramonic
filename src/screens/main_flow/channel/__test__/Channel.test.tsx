import { render } from '@testing-library/react';
import Channel from '../Channel';

describe('Channel screen', () => {
  it('should render correctly', () => {
    const { container } = render(<Channel />);

    expect(container).toMatchSnapshot();
  });
});
