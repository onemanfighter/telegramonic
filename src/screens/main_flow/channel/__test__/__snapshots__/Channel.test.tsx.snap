// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Channel screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Channel
    </h1>
  </div>
</div>
`;
